src/views/ingresses/details/components/IngressDetailsPageTitle/IngressDetailsPageTitle.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+
+import { IngressModel, modelToRef } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiNetworkingV1Ingress } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import { useLastNamespacePath } from '@utils/hooks/useLastNamespacePath';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import { getName } from '@utils/resources/shared';
+import IngressActions from '@views/ingresses/actions/IngressActions';
+
+import './IngressDetailsPageTitle.scss';
+
+type IngressDetailsPageTitleProps = {
+  ingress: IoK8sApiNetworkingV1Ingress;
+};
+
+const IngressDetailsPageTitle: FC<IngressDetailsPageTitleProps> = ({ ingress }) => {
+  const { t } = useNetworkingTranslation();
+  const namespacePath = useLastNamespacePath();
+
+  return (
+    <div className="co-m-nav-title co-m-nav-title--detail">
+      <div className="ingress-details-breadcrumbs">
+        <Breadcrumb className="pf-c-breadcrumb co-breadcrumb">
+          <BreadcrumbItem>
+            <Link to={`/k8s/${namespacePath}/${modelToRef(IngressModel)}`}>{t('Ingresses')}</Link>
+          </BreadcrumbItem>
+          <BreadcrumbItem>{t('Ingress details')}</BreadcrumbItem>
+        </Breadcrumb>
+      </div>
+      <span className="co-m-pane__heading">
```

Comment:
`Title` PF component could be more appropriate here, so you wouldn't have to add all those classes to divs and spans. Or does not using Title have some specific reason? 🤔

Reply:
Done!